packages/engine/src/interaction/components/InteractableComponent.ts

```diff
@@ -272,6 +273,8 @@ export const InteractableComponent = defineComponent({
     if (json.label) component.label.set(json.label)
     if (typeof json.uiActivationType === 'number' && component.uiActivationType.value !== json.uiActivationType)
       component.uiActivationType.set(json.uiActivationType)
+    if (typeof json.canInteract === 'boolean' && component.canInteract.value !== json.canInteract)
+      component.canInteract.set(json.canInteract)
```

Comment:
I don't think this is hurting much, but I don't think we need to serialize canInteract. It should always represent a current state.

Reply:
we need to serialize it so the user can check/uncheck the can interact editor ui checkbox, as well as loading the saved value when the scene is loaded